docker-compose-confluent-cloud.yml

```diff
@@ -11,6 +11,8 @@ services:
       CONFLUENT_SECRET: ${CONFLUENT_SECRET}
       PPM_CONFIG_FILE: ppmBsm.properties
       REDACTION_PROPERTIES_PATH: ${REDACTION_PROPERTIES_PATH}
+      PPM_LOG_TO_FILE: ${PPM_LOG_TO_FILE}
```

Comment:
Here we're updating to include the logging to file/console environment variables, but we haven't updated the sample.env to show these variables.

Reply:
I stand corrected, I expected the new variables to be at the end of the file but there are in the middle of the sample.env. Let's default them to log only to console.